ui/feed_mark_as_read.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 Frédéric Guillot. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package ui // import "miniflux.app/ui"
+
+import (
+	"net/http"
+
+	"miniflux.app/http/request"
+	"miniflux.app/http/response/html"
+	"miniflux.app/http/route"
+	"miniflux.app/logger"
+)
+
+func (h *handler) markFeedAsRead(w http.ResponseWriter, r *http.Request) {
+	feedID := request.RouteInt64Param(r, "feedID")
+
+	if err := h.feedHandler.MarkFeedAsRead(request.UserID(r), feedID); err != nil {
+		logger.Error("[UI:MarkFeedAsRead] %v", err)
```

Comment:
It might be better to generate a 500 error with `html.ServerError(....` and add a `return` after to stop the execution here.